python/delta_sharing/delta_sharing.py

```diff
@@ -93,7 +93,52 @@ def load_as_spark(url: str) -> "PySparkDataFrame":  # noqa: F821
         "No active SparkSession was found. "
         "`load_as_spark` requires running in a PySpark application."
     )
-    return spark.read.format("deltaSharing").load(url)
+    df = spark.read.format("deltaSharing")
+    if version is not None:
+        df.option("versionOf", version)
+    return df.load(url)
+
+
+def load_table_changes_as_spark(
+    url: str,
+    starting_version: Optional[int] = None,
+    ending_version: Optional[int] = None,
+    starting_timestamp: Optional[str] = None,
+    ending_timestamp: Optional[str] = None
+) -> "PySparkDataFrame":  # noqa: F821
+    """
+    Load the table changes of a shared table as a Spark DataFrame using the give url.
+    `PySpark` must be installed, and the application must be a PySpark application with
+    the Apache Spark Connector for Delta Sharing installed.
+
+    :param url: a url under the format "<profile>#<share>.<schema>.<table>"
+    :param starting_version: The starting version of table changes
+    :param ending_version: The ending version of table changes
+    :param starting_timestamp: The starting timestamp of table changes
+    :param ending_timestamp: The ending timestamp of table changes
```

Comment:
Consider adding period at the each of each line?

Reply:
added.